keras_retinanet/layers/_misc.py

```diff
@@ -174,7 +174,10 @@ def compute_output_shape(self, input_shape):
         return input_shape[0]
 
     def get_config(self):
-        return {
+        config = super(RegressBoxes, self).get_config()
```

Comment:
This doesn't seem to fit with the feature theme of FPN split output w/ Documentation.

Reply:
True, and it doesn't break anything, but it was technically incorrect. Didn't feel it deserved its own PR but maybe I should have just pushed it to master directly instead.